src/crewai/llm.py

```diff
@@ -568,6 +598,49 @@ def _handle_streaming_response(
             )
             raise Exception(f"Failed to get streaming response: {str(e)}")
 
+    def _handle_streaming_tool_calls(
+        self,
+        tool_calls: List[Any],
```

Comment:
should this be type `ToolCall`?

Reply:
Good call!

Reply:
actually, `tool_calls` comes from `LiteLLM` so it should be list of `ChatCompletionDeltaToolCall`